pkg/config/filter.go

```diff
@@ -108,3 +145,22 @@ func excludeFile(pattern string, filename string) (bool, error) {
 
 	return false, nil
 }
+
+// Taken from OPA, but while we use a different error type there, we'll settle
+// for a plain array here until we need something more fancy elsewhere.
+func errorsToString(e []error) string {
+	if len(e) == 0 {
+		return "no error(s)"
+	}
+
+	if len(e) == 1 {
+		return "1 error occurred during loading: " + e[0].Error()
+	}
+
+	buf := make([]string, len(e))
+	for i := range buf {
+		buf[i] = e[i].Error()
+	}
+
+	return fmt.Sprintf("%v errors occurred during loading:\n", len(e)) + strings.Join(buf, "\n")
```

Comment:
[nit] Inconsequential, but I've played with it so I might as well post the comment 🙃 
```suggestion
	buf := make([]string, len(e)+1)
	buf[0] = fmt.Sprintf("%v errors occurred during loading:", len(e))
	for i := range buf {
		buf[i+1] = e[i].Error()
	}

	return strings.Join(buf, "\n")
```